src/main/java/com/jfrog/ide/idea/scan/SingleDescriptorScanner.java

```diff
@@ -62,11 +62,16 @@ protected List<FileTreeNode> groupDependenciesToDescriptorNodes(Collection<Depen
         DescriptorFileTreeNode fileTreeNode = new DescriptorFileTreeNode(descriptorFilePath);
         for (DependencyNode dependency : depScanResults) {
             boolean directDep = false;
-            for (String parentId : parents.get(dependency.getComponentIdWithoutPrefix())) {
-                DepTreeNode parent = depTree.nodes().get(parentId);
-                if (descriptorFilePath.equals(parent.getDescriptorFilePath())) {
-                    directDep = true;
-                    break;
+            if(dependency.getComponentIdWithoutPrefix().equals(depTree.rootId())) {
+                directDep = true;
+            }
+            else {
+                for (String parentId : parents.get(dependency.getComponentIdWithoutPrefix())) {
+                    DepTreeNode parent = depTree.nodes().get(parentId);
+                    if (descriptorFilePath.equals(parent.getDescriptorFilePath())) {
+                        directDep = true;
+                        break;
```

Comment:
```suggestion
            if(dependency.getComponentIdWithoutPrefix().equals(depTree.rootId())) {
                dependency.setIndirect(false);
                continue
            }
            for (String parentId : parents.get(dependency.getComponentIdWithoutPrefix())) {
                    DepTreeNode parent = depTree.nodes().get(parentId);
                    if (descriptorFilePath.equals(parent.getDescriptorFilePath())) {
                        directDep = true;
                        break;
                
```

try to remove the `else`, keep code 'flat'